changelogs/client_server/newsfragments/1201.feature

```diff
@@ -0,0 +1 @@
+Add a `.m.rule.room.server_acl` push rule to match `m.room.server_acl` events, as per [MSC3786](https://github.com/matrix-org/matrix-spec-proposals/pull/3786).
```

Comment:
same changelog as https://github.com/matrix-org/matrix-spec/pull/1190 so it collapses nicely